gradoop-flink/src/main/java/org/gradoop/flink/model/api/operators/GraphCollectionOperators.java

```diff
@@ -295,4 +304,13 @@ GraphCollection apply(
    * @return graph transactions representing the graph collection
    */
   GraphTransactions toTransactions();
+
+  /**
+   * Groups a graph collection by isomorphism including labels and values.
+   *
+   * @param distinctionFunction function to generate the final group representative graph head.
+   *
+   * @return grouped graph collection
+   */
+  GraphCollection groupBy(DistinctionFunction distinctionFunction);
```

Comment:
Furthermore, if this is the expected behaviour, why don't you name the method `distinctByIsomorphism(DistinctionFunction func)` ?